lib/generators/sorcery/templates/initializer.rb

```diff
@@ -502,23 +503,23 @@
     #
     # user.authentications_class =
 
-    # User's identifier in authentications class.
+    # User's identifier in the `authentications` class.
     # Default: `:user_id`
     #
     # user.authentications_user_id_attribute_name =
 
-    # Provider's identifier in authentications class.
+    # Provider's identifier in the `authentications` class.
     # Default: `:provider`
     #
     # user.provider_attribute_name =
 
-    # User's external unique identifier in authentications class.
+    # User's external unique identifier in the `authentications` class.
     # Default: `:uid`
     #
     # user.provider_uid_attribute_name =
   end
 
   # This line must come after the 'user config' block.
   # Define which model authenticates with sorcery.
-  config.user_class = '<%= model_class_name %>'
+  config.user_class = "<%= model_class_name %>"
```

Comment:
Personally I prefer single quotes for strings that do not require interpolation. Not a blocking issue however.